docs/dev/adding_instruments.rst

```diff
@@ -510,6 +510,31 @@ Similar to `set_process` the :func:`Instrument.control <pymeasure.instruments.In
     >>> extreme.current
     3.1
 
+Another use-case of `set-process`, `get-process` is conversion from/to a :code:`pint.Quantity`. Modifying above example to set or return a quantity, we get:
+
+.. testcode::
+
+    from pymeasure.units import ureg
+
+    Extreme5000.current = Instrument.control(
+        ":CURR?", ":CURR %g",
+        """ A floating point quantity property representing the measurement current 
+        """,
+        values=[0, 10],
+        set_process=lambda v: v.m_as(ureg.mA),  # send the value as mA to the device
+        get_process=lambda v: ureg.Quantity(v, ureg.mA),  # convert to quantity
+
```

Comment:
Missing a closing parens. BTW, you can run "make doctest" in der docs directory locally, too.

Reply:
Thanks

Reply:
ah yeah, sure.
I don't notice such anymore, my pymeasure environment has been accumulating packages for quite some time. ^^
Are you using conda or pip for environment managing?

Reply:
ah yeah, sure.
I don't notice such anymore, my pymeasure environment has been accumulating packages for quite some time. ^^
Are you using conda or pip for environment managing?

Reply:
pip. Anaconda made problems with packages etc, such that I changed to pip.